ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/ShowQueriesMultiNodeFunctionalTest.java

```diff
@@ -59,11 +57,13 @@
       .builder(TEST_HARNESS::kafkaBootstrapServers)
       .withProperty(KsqlRestConfig.LISTENERS_CONFIG, "http://localhost:8088")
       .withProperty(KsqlRestConfig.ADVERTISED_LISTENER_CONFIG, "http://localhost:8088")
+      .withEnabledKsqlClient()
```

Comment:
What changed in this PR that this now needs to be specified?

Reply:
This is because after rebasing with master the branch started failing tests. I looked into this and it seems there was a change recently where a DisabledKsqlClient is not used by default in tests, so any tests that do node-node communication are going to fail (i.e. this one). I don't really know why this test didn't fail in master too. Perhaps @AlanConfluent has an insight? But explicitly requesting a non disabled KsqlClient seemed to fix things.